src/pages/components/accordion/accessibility.mdx

```diff
@@ -7,423 +7,88 @@ description:
-            </OrderedList>
-          </StructuredListCell>
-        </StructuredListRow>
-      </StructuredListBody>
-    </StructuredListWrapper>
-  </Column>
+<Column colLg={8}>
+
+![expanded and collapsed accordions](images/accordion-accessibility-2.png)
+
+</Column>
 </Row>
 
-### iOS screen reader tests
+Carbon handles the accessibility of the chevron  indicators.
+
+## Design annotations
+
+Design annotations are needed for the following instances.
+
+### Headings
+
+Carbon accordions are not set as headings by default. For improved accessibility, annotate accordions as headings on the first occurrence in a product. Annotate the heading level of accordions as needed. See [Indicate heading levels](https://www.ibm.com/able/toolkit/design/content/text-meaning/#heading-levels).
 
 <Row>
-  <Column noGutterSm>
-    <StructuredListWrapper>
-      <StructuredListRow head>
-        <StructuredListCell head>Environment </StructuredListCell>
-        <StructuredListCell head>Results</StructuredListCell>
-      </StructuredListRow>
-      <StructuredListBody>
-        <StructuredListRow>
-          <StructuredListCell>
-            - iOS version 13.1.3 with VoiceOver
-            <br />
-            - Safari version 13.1.3
-            <br />- Carbon React version 7.7.1
-          </StructuredListCell>
-          <StructuredListCell>
-            <strong>VoiceOver test:</strong>
-            <OrderedList>
-              <ListItem>
-                To navigate the accordion, swipe left or right, or use the Left
-                or Right Arrow keys when quick nav mode is on. On each header,
-                VoiceOver announces the header, and expanded/collapsed status.
-              </ListItem>
-              <ListItem>
-                To expand or collapse the accordion, doubletap on the header, or
-                press Control+Option+Space. VoiceOver announces, expanded or
-                collapses.
-              </ListItem>
-              <ListItem>
-                In some cases, VoiceOver also reads out the entire header, it
-                seems to happen randomly, possibly have to do with the speed of
-                the phone.
-              </ListItem>
-              <ListItem>
-                Navigate the text by swiping left or right, or by pressing the
-                Left or Right Arrow keys when quick navigation mode is on. After
-                each title, when moving to the left or right, VoiceOver will
-                read the text if it is expanded.
-              </ListItem>
-            </OrderedList>
-          </StructuredListCell>
-        </StructuredListRow>
-      </StructuredListBody>
-    </StructuredListWrapper>
-  </Column>
+<Column colLg={8}>
+
+![H1 and H2 annotations for headings, plus direction to ‘include accordion titles as headings'](images/accordion-accessibility-3.png)
+
+</Column>
 </Row>
 
-### Android screen reader tests
+If accordion titles act as headings, annotate for development.
```

Comment:
Is this an image caption? If so it needs to be wrapped in the caption component.
```suggestion
<Caption> If accordion titles act as headings, annotate for development. </Caption>
```

Reply:
The html `<caption>` element defines a table caption. It was our intent to use `<figcaption>`, but it didn't seem to be implemented in mdx files, so I left out for now.
I have an issue open on that

Reply:
BTW, we _really did_ have this called out already :)
![image](https://user-images.githubusercontent.com/14298245/157551980-18296812-5c45-484e-b7c6-2198cdbc5755.png)

Reply:
hmm ok well we use that caption across all our pages on the website already. There spacing is off just enough that they don't really look like captions always when just using aregular text line. That's unfortunate.

Reply:
Does it not work as a proper image caption if you put it inside the image component like:

```
<Row>
<Column colLg={8}>

![UI shell right panel example.](images/right-panel-style-1.png)

<Caption>UI shell switcher example</Caption>

</Column>
</Row>
```

Reply:
@joshblack seems like a big problem is all this is true for the website. That means all our captions across the site are wrong.

Reply:
@aagonzales what I'm seeing is that, when captions appear under images, Carbon has tended to give the image the caption as both the alt and the title, then has just placed the caption underneath in a paragraph (with some styling).